packages/tgpu-cli/tgpu-cli.mjs

```diff
@@ -11,18 +14,24 @@ const args = arg({
   '--help': Boolean,
   '--input': String,
   '--output': String,
+  '--commonjs': Boolean,
+  '--js': Boolean,
```

Comment:
Could we provide a way to configure the output extension, and infer whether to generate TypeScript or JavaScript from that? Just added a task for it #427 

```suggestion
  '--ext': String,
```